samples/trumpify/trump.py

```diff
@@ -0,0 +1,83 @@
+import os
+import sys
+import time
+from threading import Thread
+
+from twitter import *
+
+sys.path.append(os.path.abspath('../../'))
```

Comment:
Right now, Daemo Client is not published to pip so it has to be added to path to be called from the script. Once added to pip, it can be installed directly to python path.